general.go

```diff
@@ -0,0 +1,45 @@
+package flagsfiller
+
+/*
+The code in this file could be opened up in future if more complex implementation is needed
+*/
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+)
+
+// this is a list of supported struct, like time.Time, that walkFields() won't walk into,
+// the key is the is string returned by the getTypeName(<struct_type>),
+// each supported struct need to be added in this map in init()
+var supportedStructList = make(map[string]handlerFunc)
```

Comment:
This needs a new name. Something like `extendedTypes`?

Reply:
> Just the one little change and it's good to go. Thanks!

done, also updated readme